services/src/main/java/org/keycloak/jose/jws/DefaultTokenManager.java

```diff
@@ -129,6 +135,7 @@ public String signatureAlgorithm(TokenCategory category) {
             case ACCESS:
                 return getSignatureAlgorithm(OIDCConfigAttributes.ACCESS_TOKEN_SIGNED_RESPONSE_ALG);
             case ID:
+            case LOGOUT:
```

Comment:
Would be the case to have a specific attribute to configure logout signatures instead of re-using the same attribute for ID tokens?

For me, looks like what you did is OK where the client should already support any signature set for ID tokens already.

Reply:
We used the ID token implementation because in the specification it says that the logout token should be encrypted and validated similar to the id token. In particular 2.6 "Logout Token Validation" it describes the use of the encryption, used for the ID token during client registration. So we tried to reuse the current id token implementation parts instead of adding new ones.